pkgs/tools/misc/opentimestamps-client/default.nix

```diff
@@ -22,5 +22,6 @@ buildPythonApplication rec {
     description = "Command-line tool to create and verify OpenTimestamps proofs";
     homepage = "https://github.com/opentimestamps/opentimestamps-client";
     license = lib.licenses.lgpl3;
+    maintainers = with maintainers; [ erikarvstedt ];
```

Comment:
This line does not evaluate. Replace 

```
meta = {
```

above with

```
meta = with lib; {
```

(you can also refactor `license` after this change).